ui/components/app/add-network/add-network.js

```diff
@@ -273,12 +273,12 @@ const AddNetwork = () => {
                   : history.push(ADD_NETWORK_ROUTE);
               }}
             >
-              <Typography
-                variant={TypographyVariant.H6}
+              <Text
+                variant={TextVariant.bodySm}
                 color={TextColor.primaryDefault}
               >
                 {t('addANetworkManually')}
-              </Typography>
+              </Text>
             </Button>
           </Box>
         </Box>
```

Comment:
Would be very helpful to have before/after screenshots of these changes to do a visual check. I find it quite easy if there is a story for the component. Then I can use my local `yarn storybook` and compare it with the `develop` branch storybook which is hosted here https://metamask.github.io/metamask-storybook/?path=/story/components-app-addnetwork--default-story

### Before

<img width="1218" alt="Screenshot 2023-02-09 at 1 02 39 PM" src="https://user-images.githubusercontent.com/8112138/217938630-9fd103c8-b7b1-41e3-a6f4-f0b4264b0744.png">

### After

<img width="1235" alt="Screenshot 2023-02-09 at 1 02 04 PM" src="https://user-images.githubusercontent.com/8112138/217938674-677f9d9c-7ce1-4f9f-b8c4-aa88a0d28662.png">